sources/file-system/src/main/java/io/smallrye/config/source/file/FileSystemConfigSourceFactory.java

```diff
@@ -10,16 +12,17 @@
 import io.smallrye.config.ConfigValue;
 
 public class FileSystemConfigSourceFactory implements ConfigSourceFactory {
-    public static final String SMALLRYE_CONFIG_SOURCE_FILE_LOCATION = "smallrye.config.source.file.location";
+    public static final String SMALLRYE_CONFIG_SOURCE_FILE_LOCATIONS = "smallrye.config.source.file.locations";
 
     @Override
     public Iterable<ConfigSource> getConfigSources(final ConfigSourceContext context) {
-        final ConfigValue value = context.getValue(SMALLRYE_CONFIG_SOURCE_FILE_LOCATION);
+        final ConfigValue value = context.getValue(SMALLRYE_CONFIG_SOURCE_FILE_LOCATIONS);
         if (value == null || value.getValue() == null) {
             return Collections.emptyList();
         }
-
-        return Collections.singletonList(new FileSystemConfigSource(value.getValue()));
+		return Stream.of(value.getValue().split(","))
```

Comment:
Try using here `newArrayConverter(STRING_CONVERTER, String[].class).convert(value.getValue())`

Reply:
Ops. I guess it is ok to add a new method to retrieve the a Converter by Type.

Reply:
Ok will change it next week